QUEST_LV_0100.tsv

```diff
@@ -6702,7 +6702,7 @@ QUEST_LV_0100_20150428_006701	If you have time, please tell our troops to assemb
 QUEST_LV_0100_20150428_006702	Thank you so much for your support.{nl}Follow the road on the left of the camp crossroads. {-scp SCR_DIALOG_NPC_ANIM(
 QUEST_LV_0100_20150428_006703	{memo X}Vubbe Fighter is hiding deep in the Nudegi Felled Area. {nl} We got the orders to kill but it may be safer to wait for additional troops.
 QUEST_LV_0100_20150428_006704	We should check by rubbing Hallowventer's wooden coals.{nl}The next epitaph is at the right side of Negibas Fields.
-QUEST_LV_0100_20150428_006705	We can suppress the vicious energy if the magic of the tomb get unleashed.{nl}I need the activating stone that is located inside the protector.
+QUEST_LV_0100_20150428_006705	$We can suppress the evil forces if the magic of the tomb is released.{nl}This requires a number of activating stones located inside protectors.
```

Comment:
of the tomb -> of the Royal Mausoleum
protectors -> the guardians (the guardians?)